tests/dummy/app/templates/demos/sl-modal.hbs

```diff
@@ -0,0 +1,179 @@
+<h2>sl-modal</h2>
+<p class="lead">A modal component used to wrap content in. Based on <a href="http://getbootstrap.com/javascript/#modals">Bootstrap - Modals</a>.</p>
+<hr>
+
+<h3>Examples</h3>
+
+<section id="simple">
+    <div class="row">
+        <div class="col-lg-6">
+            <h6>Simple usage</h6>
+            <pre>\{{#sl-modal name="simpleDemo}}
+    \{{sl-modal-header title="Simple Example"}}
+
+    \{{#sl-modal-body}}
+         &lt;p&gt;A simple modal example&lt;/p&gt;
+    \{{/sl-modal-body}}
+
+    \{{sl-modal-footer}}
+\{{/sl-modal}} </pre>
+        {{#sl-modal name="simpleDemo"}}
+                {{sl-modal-header title="Simple Example"}}
+
+                {{#sl-modal-body}}
+                    <p>A simple modal example</p>
+                {{/sl-modal-body}}
+
+                {{sl-modal-footer}}
+            {{/sl-modal}}
+        </div>
+        <div class="col-lg-6">
+            <h6>Click to Show Component</h6>
+            {{sl-button openModal="simpleDemo" label="Open Modal"}}
+        </div>
+    </div>
+</section>
+
+<section>
+    <div class="row">
+        <div class="col-lg-6">
+            <h6>Custom header</h6>
+            <pre>\{{#sl-modal name="customHeader"}}
+    \{{sl-modal-header}}
```

Comment:
`#` hints that it's a block, in the case above it's inline and there's no need for a block hence the `#` is left off.